h/static/styles/partials-v2/_share-widget.scss

```diff
@@ -0,0 +1,138 @@
+.share-widget-template {
+  visibility: hidden;
+  // we want template to have no effect
+  // on the page layout, but keep it in it's
+  // current flow
+  height:0;
+  width:0;
+}
+
+.share-widget.is-open {
+  display: flex;
+  flex-direction: column;
+}
+
+.share-widget {
```

Comment:
I presume much of this is adapted from the client's styling. It would be useful to have some brief (even just one line) comments above the main classes in this file indicating elements of the share dialog they are.